webpack.config.js

```diff
@@ -294,7 +294,10 @@ export default async (env, argv) => {
     const plainAliases = getIconPlainAliases(iconsDataBySlugs[icon.slug]);
 
     return {
-      guidelines: icon.guidelines,
+      guidelines:
+        typeof icon.guidelines != 'object'
```

Comment:
Should we not be using a "not strictly equal to" check here?
```suggestion
        typeof icon.guidelines !== 'object'
```